node/src/db_config/config_dao.rs

```diff
@@ -287,16 +291,23 @@ mod tests {
     }
 
     #[test]
-    fn setting_by_started_transaction_works() {
+    fn set_by_guest_transaction_works() {
         let home_dir =
-            ensure_node_home_directory_exists("config_dao", "setting_by_started_transaction_works");
-        // To make the test even more convincing, let's create a useless "Connection".
+            ensure_node_home_directory_exists("config_dao", "set_by_guest_transaction_works");
+        // To make the test even more convincing, let's give a useless "Connection" to the hosting DAO object,
+        // while the guest connection will do the job
         let subject = ConfigDaoReal::new(Box::new(ConnectionWrapperMock::default()));
         let mut conn = initialize_conn(&home_dir);
```

Comment:
It creates the new database with the version number according to the constant `CURRENT_SCHEMA_VERSION`. You wanted to make it a part of this test as a comment.